Fixes the plugin crash when version field is not available. (#104)
logstash-plugins/logstash-codec-cef: CHANGELOG.md

## 6.2.8

>  - Fixes the plugin crash when version field is not available [#104](https://github.com/logstash-plugins/logstash-codec-cef/pull/104)

Reviewer: I would suggest to use the other changelogs format:
```suggestion
 - Fix: avoid to crash when version field is not available [#104](https://github.com/logstash-plugins/logstash-codec-cef/pull/104)
```
